packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -580,3 +580,110 @@ class _UiKitPlatformView extends LeafRenderObjectWidget {
     renderObject.updateGestureRecognizers(gestureRecognizers);
   }
 }
+
+/// Embeds an existing platform view in the widget tree.
+///
+/// A "PlatformView" is a view that being displayed with the help of the platform's UI system
```

Comment:
nit: A "platform view" is a UI component of the platform Flutter is running on; for example on Android `android.view.View`s are platform views.

Reply:
done